target/linux/qualcommbe/files/arch/arm64/boot/dts/qcom/ipq9574-route10.dts

```diff
@@ -0,0 +1,515 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause)
+
+/dts-v1/;
+
+#include "ipq9574.dtsi"
+
+#include <dt-bindings/clock/qcom,ipq-cmn-pll.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/clock/qcom,qca8k-nsscc.h>
+#include <dt-bindings/net/qcom,qca808x.h>
+#include <dt-bindings/reset/qcom,qca8k-nsscc.h>
+
+/ {
+	model = "Alta Labs Route10";
+	compatible = "alta,route10", "qcom,ipq9574";
+
+	aliases {
+		serial0 = &blsp1_uart2;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	qca8k_uniphy1_rx312p5m: qca8k-uniphy1-rx312p5m {
+		compatible = "fixed-clock";
+		clock-frequency = <312500000>;
+		#clock-cells = <0>;
+	};
+
+	qca8k_uniphy1_tx312p5m: qca8k-uniphy1-tx312p5m {
+		compatible = "fixed-clock";
+		clock-frequency = <312500000>;
+		#clock-cells = <0>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-0 = <&sfp_leds>;
+		pinctrl-names = "default";
+
+		led-0 {
+			color = <LED_COLOR_ID_WHITE>;
+			label = "led10g_p5";
+			gpios = <&tlmm 61 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```

Comment:
`default-state = "off"` is the default state and can be removed

Reply:
removed for all 4 LED's